packages/package.go

```diff
@@ -76,7 +88,9 @@ func LoadPackageFromPath(path string) (*Package, error) {
 	if err != nil {
 		return nil, err
 	}
-	pkg := &Package{}
+	pkg := &Package{
+		WorkingDir: "/workdir",
```

Comment:
Yes, true, compose also uses working_directory in the yaml description
It is probably better to use the go syntax, I will update the PR